scripts/install_operator.sh

```diff
@@ -25,10 +25,7 @@ else
       labels:
         name: "${NAMESPACE}"
 EOF
-    # For the time being we need to install CRDs manually because KUDO doesn't fully support it.
-    # Once the issue will be resolved https://github.com/kudobuilder/kudo/issues/935,
-    # move the CRDs instalaltion back to the kudo-spark-operator.
-    kubectl apply -f ${SPECS_DIR}/spark-applications-crds.yaml
```

Comment:
can you please remove that from scripts in `scale-tests`  too?

Reply:
Sure, I will do that.
But gotcha :-), happened to what about I wrote there https://github.com/mesosphere/kudo-spark-operator/pull/22#discussion_r335417217

Reply:
Thanks for the reminder. There are some reasoning and context behind the scale tests scripting to make them completely independent from any other scripts in this repo thus portable. Same applies to the tests - they should be fully autonomous and rely on API and program modules instead of external helper scripts created as a workaround for the missing from the underlying technology features. DRY principle, in this case, introduces more problems in the long term then solves in the short term.